modules/distr-tracing-tempo-deploy-default.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-deploy-default_{context}"]
+= Deploying the {DTShortName} default strategy from the web console
+
+The custom resource definition (CRD) defines the configuration used when you deploy an instance of {DTProductName}. The default CR is configured with minimal resources to ensure that you can successfully install it on a default {product-title} installation. You can use this default configuration to create a basic {TempoName} instance, or you can define your own custom resource file.
+
+[NOTE]
+====
+In-memory storage is not persistent. If the Tempo pod shuts down, restarts, or is replaced, your trace data will be lost. For persistent storage, you must use the `production` or `streaming` strategies, which use Elasticsearch as the default storage.
+====
+
+.Prerequisites
+
+* The {TempoName} Operator has been installed.
+* You have reviewed the instructions for how to customize the deployment.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* Supported Object Store (AWS S3, Google Cloud Storage, Azure, Minio, OpenShift Data Foundation)
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with the `cluster-admin` role.
+
+. Create a new project, for example `tracing-system`.
++
+[NOTE]
+====
+If you are installing as part of Service Mesh, the {DTShortName} resources must be installed in the same namespace as the `ServiceMeshControlPlane` resource, for example `istio-system`.
+====
++
+.. Navigate to *Home* -> *Projects*.
```

Comment:
I would add a note here like this:

[NOTE]
====
Before the creation of the tempoStack instance  you need to create a secret that will contain the storage backend configuration.
====